src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -22,25 +35,35 @@
   private final Joystick m_driverJoystick;
   private final MotorController[] m_motorControllers;
   private final DifferentialDrive m_differentialDrive;
-  //public static ADIS16448_IMU m_gyro; Non-native gyro, might use later
-  public static ADXRS450_Gyro m_gyro;
+  private DifferentialDrivetrainSim m_DifferentialDrivetrainSim;
+  public final Field2d m_field = new Field2d();
+  private AnalogGyro m_gyro;
+  private AnalogGyroSim m_gyroSim;
+  public static ADIS16448_IMU m_gyro2; //Non-native gyro, might use later
+  public static ADXRS450_Gyro m_gyro1;
   private final DifferentialDriveOdometry m_odometry;
   public static Encoder m_leftEncoder;
   public static Encoder m_rightEncoder;
+  private EncoderSim m_leftEncoderSim;
+  private EncoderSim m_rightEncoderSim;
+  
 
   // Here are the encoders
   
   
 
-  public DriveBaseSubsystem(Joystick joystick) {  
+  public DriveBaseSubsystem(Joystick joystick) { 
     m_leftEncoder = new Encoder(Constants.leftEncoderDIOone, Constants.leftEncoderDIOtwo, 
-    false, Encoder.EncodingType.k2X);
+                                false, Encoder.EncodingType.k2X);
+
     m_rightEncoder = new Encoder(Constants.rightEncoderDIOone, Constants.rightEncoderDIOtwo, 
-    false, Encoder.EncodingType.k2X);
+                                false, Encoder.EncodingType.k2X);
+
     m_driverJoystick = joystick;
+
     m_motorControllers = new MotorController[4];
-    m_gyro = new ADXRS450_Gyro();
-    m_odometry = new DifferentialDriveOdometry(m_gyro.getRotation2d());
+    m_gyro1 = new ADXRS450_Gyro();
```

Comment:
Why the rename?

Reply:
Idk but was not needed to I got rid of it